tests/pkg-spine-order-svg/OPS/nav.xhtml

```diff
@@ -0,0 +1,12 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>SVG Spine Items in Order</title>
```

Comment:
Nit: sentence case in all files, per https://w3c.github.io/epub-tests/contributing#metadata

Reply:
Fixed. Title case for titles is an occupational hazard :)